api/v1alpha3/webhook_test.go

```diff
@@ -30,6 +30,8 @@ import (
 )
 
 func TestClusterConversion(t *testing.T) {
+	t.Skipf("v1alpha3 is no longer served see: https://cluster-api.sigs.k8s.io/contributing#removal-of-v1alpha3--v1alpha4-apiversions")
```

Comment:
I would prefer deleting the tests. It's not like we're just skipping them temporarily :)

(same in other cases)